qubes/vm/dispvm.py

```diff
@@ -624,118 +703,121 @@ async def from_appvm(
         >>> dispvm.cleanup()
 
         This method modifies :file:`qubes.xml` file.
-        The qube returned is not started unless the ``preload`` argument is
-        ``True``.
+        """
+        if not DispVM.can_gen_disposable(appvm, preload=preload):
```

Comment:
```suggestion
        if not cls.can_gen_disposable(appvm, preload=preload):
```
and similar in few places below.